index.js

```diff
@@ -150,12 +151,12 @@ app.delete('/users/:username/favorites', (req, res) => {
 
     // If user exists and they have the film as a favorite
     if (user) {
-        let isFavorite = user.favorites.find((favorite) => { return favorite.title === notFavoriteAnymore.title });
+        let isFavorite = user.favorites.find((favorite) => { return favorite.title === notFavoriteAnymore });
         if (isFavorite) {
-            user.favorites.delete(notFavoriteAnymore);
-            res.status(201).send(`You removed ${notFavoriteAnymore.title} as a favorite.`);
+            user.favorites = user.favorites.filter((obj) => { return obj.title !== notFavoriteAnymore });
+            res.status(201).send(`You removed ${notFavoriteAnymore} as a favorite.`);
```

Comment:
The status code for deleted is `204` that actually returns no content. `201` is for created successfully. Please take a look [here](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status) on when to use a specific status code.